website/pages/bunny/security.mdx

```diff
@@ -0,0 +1,65 @@
+
+Bunny should be deployed using container images to ensure consistency and security. 
+
+### Container Image Build Process
+- Bunny images are built using GitHub Actions, using a repeatable and transparent build process. 
+- Each dependency from the build process is pinned to a git hash.
+- This automation ensures that each build is consistent and can be traced back to its source code and build instructions.
+- Organisations can build their own image from source if required.
+
+### Container Registry
+- Every Bunny image is published to the GitHub container registry. 
+- Images are pinned to specific workflow commit hashes, which prevents unauthorized modifications and ensures that only verified builds are used. 
+- This pinning mechanism adds security by ensuring that the exact version of the code that was reviewed and tested is the one being deployed.
+
+### Security Reviews and Updates
+- The base image and all dependencies are reviewed for security updates.
+- All code contributions must pass a comprehensive set of unit, integration, and end to end tests. 
+- Contributions are reviewed by the University of Nottingham Health Informatics developer team, before they are approved and merged into the codebase.
+- Code scanning using GitHub's CodeQL analysis is enabled on the repository and contributions to automatically identify potential security vulnerabilities and coding issues.
+- For vulnerability reporting, Bunny has a [security policy](https://github.com/Health-Informatics-UoN/hutch-bunny/security) in place.
+
+### Version Pinning
+- We recommend pinning deployed Bunny containers to a specific tagged version for stability. 
+- Using tagged versions ensures that you are deploying a known, tested version of the software, which helps in maintaining a stable and predictable environment. 
+- You can find a list of available version tags in the [container registry](https://github.com/Health-Informatics-UoN/hutch-bunny/pkgs/container/hutch%2Fbunny).
+
+## Infrastructure Security
+
+![Bunny infrastructure](/images/infrastructure.png)
+_An example Bunny deployment_.
+
+- Bunny is deployed in a secure environment by a data partner. 
+- Bunny runs as a containerised service and should be deployed within a private subnet for isolation from other services.
+- Bunny should be secured with network policies to restrict outbound connections to the RQUEST / Relay API.
+- Bunny makes only outgoing connections to RQUEST / Relay API over HTTPS (port 443).
+- Bunny requires access to the data partner's OMOP CDM database.
+- No incoming requests are made into the data partner's secure environment.
+
+## Authentication & Access Control
+- All credentials (database access, RQUEST / Relay authentication) are managed through secure environment variables.
+- All credentials can be stored in a secrets service (for example Azure Key Vault).
+- Bunny connections to RQUEST / Relay are secured using Basic Auth, consistent with the API specification.
```

Comment:
```suggestion
- Bunny connections to RQUEST / Relay are secured using HTTP Basic Auth, consistent with the API specification.
```

I would contextualise this as HTTP in some way. Maybe the above, maybe not.

The IETF refer to it today as

> The 'Basic' HTTP Authentication Scheme

_[RFC7617](https://datatracker.ietf.org/doc/html/rfc7617)_

And there or thereabouts in the [original spec](https://www.w3.org/Protocols/HTTP/1.0/spec.html#BasicAA) (which is how long its been around!)

Reply:
I'm going to go with the full name from the spec